src/aten/sycl/ActivationLeakyReluKernels.cpp

```diff
@@ -0,0 +1,68 @@
+#include <ATen/ATen.h>
+#include <ATen/Dispatch.h>
+#include <ATen/NumericUtils.h>
+#include <ATen/native/Activation.h>
+#include <ATen/native/TensorIterator.h>
+#include <aten/sycl/Loops.h>
+
+namespace at {
+namespace native {
+namespace xpu {
+
+template <typename scalar_t>
+struct LeakyReluFunctor {
+  scalar_t operator()(scalar_t a) const {
+    opmath_t aop = static_cast<opmath_t>(a);
```

Comment:
I m not sure of the syntax of C++ here.
Where is the forward declaration of opmath_t?